aries-site/src/data/wcag/components.json

```
@@ -3003,5 +3003,519 @@
         "status": "not-applicable"
       }
     ]
+  },
+  {
+    "checkboxgroup": [
+      {
+        "rule": "1.1.1",
+        "label": "Non-text content",
+        "status": "passed",
+        "notes": "Tested as part of checkbox + label"
+      },
+      {
+        "rule": "1.2.1",
+        "label": "Audio-only and Video-only (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.2",
+        "label": "Captions (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.3",
+        "label": "Audio Description or Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.4",
+        "label": "Captions (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.5",
+        "label": "Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.6",
+        "label": "Sign Language (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.7",
+        "label": "Extended Audio Description (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.8",
+        "label": "Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.9",
+        "label": "Audio-only (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "failed",
+        "notes": "Screenreader does no read out the number of items in a checkbox group or notifiy the user which item they are on."
```

I looked at the native html checkbox behavior and other react libraries check box group behavior, and the way the screenreader is reading the grommet CheckBoxGroup is consistent with those. I know originally I was thinking this should be marked as failing but on second thought I think it is passing. I don't think the number of items in the checkbox group is critical information for the screenreader to read out.
```suggestion
        "status": "passed",
        "notes": ""
```